src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,39 @@
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[] | null;
```

Comment:
```suggestion
  people: Person[];
```
You don't need to render `PeopleTable` if you don't have them